packages/ui/v2/core/WizardForm.tsx

```diff
@@ -18,9 +19,13 @@ function WizardForm<T extends DefaultStep>(props: {
   steps: T[];
   disableNavigation?: boolean;
   containerClassname?: string;
+  t?: TFunction;
 }) {
   const { href, steps } = props;
-  const { t } = useLocale();
+  let { t } = useLocale();
+  if (props.t) {
+    t = props.t;
+  }
```

Comment:
Making the translation configurable from outside in order to be able to use this component from different packages resolving translations differently.

Reply:
interesting, havent seen this before. is this a good pattern @zomars @emrysal ?

Reply:
I would say it’s definitely not ideal, as co sole should have official i18n support, so any cross-package ui component that needs translation can be natively translated instead of artificially like this, but as of now, unless we are ready to introduce i18n in Console, it’s just a monkey patch

Reply:
I have my doubts. Also I think WizardForm should not handle translations at all. It should receive everything pre translated.